src/wp-includes/functions.php

```diff
@@ -2074,6 +2075,16 @@ function wp_mkdir_p( $target ) {
 		$dir_perms = 0777;
 	}
 
+	/**
+	 * Fires before the directory is created
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $target Full path to attempt to create
+	 * @param int $dir_perms Directory permissions
```

Comment:
```suggestion
	 * @param string $target    Full path to attempt to create.
	 * @param int    $dir_perms Directory permissions.
```
- The types and descriptions used in `@param` annotations should be aligned.
- Parameter descriptions should end with a period.